### PR TITLE
ocioconvert/ociolutimage: Change: unique_ptr to raw ptr

### PR DESCRIPTION
**Pull request by LiamGFX**
- Visual Studio 2017 on Windows 7 x64 SP1 errors out because it can not convert the unique_ptr to a raw ptr

### CODE REVIEW

**Comment by LiamGFX** [comment]
Hi, the CI/Windows doesn't build the apps and as a result the "checks" here don't have any meaning.

Also, I'm not done 100% done compiling OCIO/Windows and plan to make more Pull Requests as I find things that are broken.

Thx! Liam

File: src/apps/ociolutimage/main.cpp [hunk 1]
```diff
@@ -305,12 +305,18 @@ int main (int argc, const char* argv[])
 // TODO: These should be exposed from inside OCIO, in appropriate time.
 //
 
-inline int GetLut3DIndex_RedFast(int indexR, int indexG, int indexB,
```
Comment (michdolan) [inline src/apps/ociolutimage/main.cpp]: The changes below this point are reverting some updates from PR #535. Can you update your branch from master and do another commit? Thanks.

File: src/apps/ocioconvert/main.cpp [hunk 2]
```diff
@@ -119,7 +119,7 @@ int main(int argc, const char **argv)
     std::cerr << "Loading " << inputimage << std::endl;
     try
     {
-        OIIO::ImageInput* f = OIIO::ImageInput::create(inputimage);
+        OIIO::ImageInput* f = (OIIO::ImageInput::create(inputimage)).get();
```
Comment (michdolan) [inline src/apps/ocioconvert/main.cpp]: OIIO's change to unique_ptr (from a raw ptr) is quite new. This change breaks the build with older versions of OIIO. You can use the OIIO_VERSION macro to conditionally handle both APIs.

File: src/apps/ociolutimage/main.cpp (hunk 1 above)
Comment (LiamGFX) [reply src/apps/ociolutimage/main.cpp]: Sorry for the oversight

File: src/apps/ocioconvert/main.cpp (hunk 2 above)
Comment (LiamGFX) [reply src/apps/ocioconvert/main.cpp]: Cool, I'll look into it. Thx!

**Comment by LiamGFX** [comment]
Closing, will create again.
